internal/storage/v2/elasticsearch/tracestore/to_dbmodel_test.go

```diff
@@ -263,108 +267,67 @@ func TestAttributesToDbSpanTags_MapType(t *testing.T) {
 	require.Equal(t, expected, got)
 }
 
-func TestToDBModel(t *testing.T) {
-	tests := []struct {
-		name string
-		td   ptrace.Traces
-		db   []dbmodel.Span
-	}{
-		{
-			name: "empty",
-			td:   ptrace.NewTraces(),
-		},
-
-		{
-			name: "no-resource-attrs",
-			td:   generateTracesResourceOnlyWithNoAttrs(),
-		},
-
-		{
-			name: "one-span-no-resources",
-			td:   generateTracesOneSpanNoResourceWithTraceState(),
-			db:   []dbmodel.Span{generateProtoSpanWithTraceState()},
-		},
-		{
-			name: "library-info",
-			td:   generateTracesWithLibraryInfo(),
-			db:   []dbmodel.Span{generateProtoSpanWithLibraryInfo("io.opentelemetry.test")},
-		},
-		{
-			name: "two-spans-child-parent",
-			td:   generateTracesTwoSpansChildParent(),
-			db: []dbmodel.Span{
-				generateProtoSpan(),
-				generateProtoChildSpan(),
-			},
-		},
-
-		{
-			name: "two-spans-with-follower",
-			td:   generateTracesTwoSpansWithFollower(),
-			db: []dbmodel.Span{
-				generateProtoSpan(),
-				generateProtoFollowerSpan(),
-			},
-		},
+func TestToDbModel_Fixtures(t *testing.T) {
+	tracesStr, spansStr := loadFixtures(t, 1)
+	var span dbmodel.Span
+	err := json.Unmarshal(spansStr, &span)
+	require.NoError(t, err)
+	td, err := FromDBModel([]dbmodel.Span{span})
+	require.NoError(t, err)
+	testTraces(t, tracesStr, td)
+	spans := ToDBModel(td)
+	assert.Len(t, spans, 1)
+	testSpans(t, spansStr, spans[0])
+}
 
-		{
-			name: "span-with-span-event-attribute",
-			td:   generateTracesOneSpanNoResourceWithEventAttribute(),
-			db:   []dbmodel.Span{generateJProtoSpanWithEventAttribute()},
-		},
-		{
-			name: "a-spans-with-two-parent",
-			td:   generateTracesSpanWithTwoParents(),
-			db: []dbmodel.Span{
-				generateProtoSpan(),
-				generateProtoFollowerSpan(),
-				generateProtoTwoParentsSpan(),
-			},
-		},
-	}
+func writeActualData(t *testing.T, name string, data []byte) {
+	var prettyJson bytes.Buffer
+	err := json.Indent(&prettyJson, data, "", "  ")
+	require.NoError(t, err)
+	err = os.WriteFile("actual_"+name+".json", prettyJson.Bytes(), 0o644)
```

Comment:
nit: I suggest writing these files under `fixtures/` directory and adding a `.gitignore` to it for `actual_*` to avoid accidentally checking them in. Also, write a log statement `t.Logf()` indicating where the file was saved.